### ISSUE TITLE
How to increase n_epoch level

### ISSUE THREAD

**Issue by sarancruzer** [ugik/notebooks#3]
# reset underlying graph data
tf.reset_default_graph()
# Build neural network
net = tflearn.input_data(shape=[None, len(train_x[0])])
net = tflearn.fully_connected(net, 8)
net = tflearn.fully_connected(net, 8)
net = tflearn.fully_connected(net, len(train_y[0]), activation='softmax')
net = tflearn.regression(net)

# Define model and setup tensorboard
model = tflearn.DNN(net, tensorboard_dir='tflearn_logs')
# Start training (apply gradient descent algorithm)
model.fit(train_x, train_y, n_epoch=1000, batch_size=8, show_metric=True)
model.save('model.tflearn')

**Comment by MauryaRitesh** [comment]
Use n_epoch = 500000 or whatever, instead of 1000. Like this:
model.fit(train_x, train_y, n_epoch=500000, batch_size=8, show_metric=True)
